profile/profile.yaml

```diff
@@ -0,0 +1,121 @@
+# Preferred name (Characters only)
```

Comment:
please run a grammar and spelling check via Grammarly. I have noticed several typos